packages/big-design/src/components/AccordionPanel/spec.tsx

```diff
@@ -0,0 +1,137 @@
+import { ErrorIcon } from '@bigcommerce/big-design-icons';
+import userEvent from '@testing-library/user-event';
+import React, { useState } from 'react';
+
+import { render, screen } from '@test/utils';
+
+import { Text } from '../Typography';
+
+import { AccordionProps } from './Accordion';
+
+import { AccordionPanel, useAccordionPanel } from '.';
+
+let panels: AccordionProps[];
+
+const TestComponent: React.FC = () => {
+  ({ panels } = useAccordionPanel([
+    {
+      header: 'Panel Header',
+      children: <Text>This is a child component</Text>,
+    },
+  ]));
```

Comment:
Maybe we should just assign this to a local const instead of using this type of JavaScript sorcery.
```suggestion
  const { panels } = useAccordionPanel([
    {
      header: 'Panel Header',
      children: <Text>This is a child component</Text>,
    },
  ]);
```
Same below...